File: Easy/C++/SumOfDigits.cpp
#include <iostream>
#include <cstdlib>
using namespace std;

int sum_of_digits(int n)
{
  n = abs(n);
  int sum = 0;
  while (n > 0)
  {
    sum += n % 10;
    n /= 10;
  }
  return sum;
}

int main()
{
  int num;
  cout << "Enter a number: ";
  cin >> num;
  cout << sum_of_digits(num) << endl;
  return 0;
}

File: Easy/Java/SumOfDigits.java
import java.util.Scanner;

public class SumOfDigits {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    System.out.print("Enter a number: ");
    int num = sc.nextInt();
    sc.close();

    int sum = 0, n = Math.abs(num);
    while (n > 0) {
      sum += n % 10;
      n /= 10;
    }

    System.out.println(sum);
  }
}
